app/api/stories/stories.controller.js

```diff
@@ -67,23 +71,28 @@ const addStory = async (req, res) => {
  * @param {*} res 
  */
 const updateStory = async (req, res) => {
-  let { title, logo_link } = req.body;
+  const { title, logo_link, description, description_preview }= req.body;
 
   try {
-    const updatedStory = await Stories.update({ title, logo_link }, {where: {id: req.params.id}});
-    res.status(201).json(updatedStory);
+    await models.stories.update({ title, logo_link, description, description_preview }, {where: {id: req.params.id}});
+    const updatedStory = await models.stories.findByPk(req.params.id);
```

Comment:
use option `{ returning: true }` to get updated value directly from `update`as in :
 ```
 const problem = await req.problem.update(
    {
      state: req.body.state
    },
    {
      returning: true
    }
  );
```